tests/integration/test_filter.py

```diff
@@ -110,6 +109,7 @@ def test_filter_cast_date(c, input_table, request):
         ),
     ],
 )
+@pytest.mark.xfail(reason="Need support for non-UTC timezoned literals")
```

Comment:
Might be worth opening and linking to an issue here if possible (but not blocking)